2023/css/styles.css

```diff
@@ -1,28 +1,41 @@
 :root {
     --base-font: 1.6rem;
-    --clr-bg: #45a480;
-    --clr-hdr: #f99c14;
+    --clr-bg: #f5f5f5   ;
+    --clr-hdr: #78B4BE;
     --clr-txt: #2c3f52;
-    --clr-ctn: #e3753f;
-    --clr-txt2: #e3d46f;
+    --clr-ctn: #78B4BE;
+    --clr-txt2: #3d3d3d;
 }
 
 * {
     margin: 0;
     padding: 0;
+    box-sizing: border-box;
```

Comment:
curious what this is for

Reply:
pretty sure that was to get the sponsors under the cards, not positive... ill look closer into it